src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -73,6 +78,11 @@ public async Task RunApplicationAsync(CancellationToken cancellationToken = defa
         AspireEventSource.Instance.DcpModelCreationStart();
         try
         {
+            if (!_model.Resources.Any(r => r.Name == "aspire-dashboard"))
```

Comment:
If this name is special, then make a constant. A better solution could be to have internal metadata to avoid accidental conflict using name.

Reply:
You mean an annotation? The problem is that the playground projects would need to be able to see that annotation.

Reply:
Made constant in #1775